package.json

```
@@ -21,13 +21,13 @@
 		"release-zip": "npm run zip",
 		"pretest": "npm run build:production -- --display=none",
 		"pretest:visualtest": "VRT=1 npm run build:storybook",
-		"test": "npm run test:js:watch",
+		"test": "npm run test:js:watch -w ./tests/js",
 		"test:visualtest": "npm run test:visualtest -w ./tests/backstop",
 		"test:visualapprove": "npm run test:visualapprove -w ./tests/backstop",
 		"test:analyze": "npm run build:production -- --env.ANALYZE",
-		"test:js": "cross-env BABEL_ENV=test NODE_ENV=test jest --config=tests/js/jest.config.js --passWithNoTests",
-		"test:js:watch": "npm run test:js -- --watch",
-		"test:storybook": "npm run test:js -- --testMatch '<rootDir>/storybook/*.test.js'",
+		"test:js": "npm run test:js -w ./tests/js",
+		"test:js:watch": "npm run test:js:watch -w ./tests/js",
+		"test:storybook": "cross-env BABEL_ENV=test NODE_ENV=test jest --config=./tests/js/jest.config.js --passWithNoTests --testMatch '<rootDir>/storybook/*.test.js'",
```

Ok, if we want to do it as part of the #10091 ticket, then we need to leave only what is really needed in this command and rework it in that ticket to have the jest dependency loaded separately from this `tests/js` workspace.
```suggestion
		"test:storybook": "jest --config=./tests/js/jest.config.js ./storybook/stories.test.js",
```

Plus, we need to leave a comment in that ticket that this part should be updated there.